Misc/NEWS.d/next/C API/2021-07-20-16-21-06.bpo-42747.rRxjUY.rst

```diff
@@ -0,0 +1,4 @@
+The ``Py_TPFLAGS_HAVE_VERSION_TAG`` type flag now does nothing. The
+``Py_TPFLAGS_HAVE_AM_SEND`` flag (which was added in 3.10) is removed. Both
```

Comment:
> ``Py_TPFLAGS_HAVE_AM_SEND`` flag (which was added in 3.10) is removed.

there aren't many users use it?

Reply:
If there any users of it, they are using the 3.10 betas. Requiring that they remove the flag is fair game.